scripts/networks/orchestrate.ts

```diff
@@ -131,6 +142,26 @@ class Loader {
     return flat;
   }
 
+  async getTokenConfig(recipe: Sections[]): Promise<ITokenConfig[]> {
+
+    const existingKeys = Object.keys(this.existing)
+    const pyroCableSections: Sections[] = [Sections.LiquidityReceiverOld, Sections.LiquidityReceiverNew, Sections.BehodlerTokens]
+
+    const isPyroCapable = pyroCableSections.every(s => recipe.includes(s))
+    console.log('isPyroCapable: ' + isPyroCapable)
```

Comment:
Maybe replace `console.log` with the defined `logger`?

Reply:
logger is for logical debugging whereas console.log means I accidentally didn't delete all the console.logs before committing.